dds/DCPS/SafetyProfileStreams.h

```diff
@@ -20,16 +20,14 @@ OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
 namespace OpenDDS {
 namespace DCPS {
 
-OpenDDS_Dcps_Export OPENDDS_STRING to_dds_string(::CORBA::UShort to_convert);
+OpenDDS_Dcps_Export OPENDDS_STRING to_dds_string(ACE_CDR::UShort to_convert);
```

Comment:
```suggestion
OpenDDS_Dcps_Export OPENDDS_STRING to_dds_string(unsigned short to_convert);
```
These aren't related to CDR.